src/main/java/org/folio/service/orders/flows/update/unopen/UnOpenCompositeOrderManager.java

```diff
@@ -169,10 +169,6 @@ private Future<Void> processInventoryOnlyWithHolding(CompositePoLine compPOL, Re
   }
 
   private Future<Void> processInventoryOnlyWithItems(CompositePoLine compPOL, RequestContext requestContext) {
-    if (!PoLineCommonUtil.isReceiptNotRequired(compPOL.getReceiptStatus())) {
-      logger.info("Skip deleting items and pieces for Un-Open order with id: {} because receipt can be required", compPOL.getId());
-      return Future.succeededFuture();
-    }
```

Comment:
Check of PoLineCommonUtil.isItemsUpdateRequired(compPOL) is made before this method invocation, so not needed here